doc/reference/reference_rock/index.rst

```diff
@@ -10,6 +10,6 @@ This reference covers third-party Lua modules for Tarantool.
     dbms
     expirationd
     membership
-    shard
     vshard/index
-    tdb
+    ../../book/cartridge/index
```

Comment:
Is it intentional?

Reply:
yes, @artur-barsegyan asked me to remove tbd and add cartridge here